backend_new/prisma/seed-helpers/listing-factory.ts

```diff
@@ -0,0 +1,388 @@
+import {
+  Prisma,
+  ApplicationAddressTypeEnum,
+  ListingsStatusEnum,
+  ReviewOrderTypeEnum,
+  MarketingTypeEnum,
+  MarketingSeasonEnum,
+  HomeTypeEnum,
+  RegionEnum,
+  ApplicationMethodsTypeEnum,
+  ListingEventsTypeEnum,
+  MultiselectQuestionsApplicationSectionEnum,
+  UnitsStatusEnum,
+} from '@prisma/client';
+
+export const listingFactory = (
+  i: number,
+  jurisdictionId: string,
+): Prisma.ListingsCreateInput => ({
+  additionalApplicationSubmissionNotes: `additionalApplicationSubmissionNotes: ${i}`,
+  digitalApplication: true,
```

Comment:
If this is just default data can these be null?

Reply:
I think eventually we generate some actual seed data and this listing-factory becomes strictly if we need to test something about large amounts of listings or integration testing data 

lemme know if that makes sense or if you'd prefer to see this data get some updates